src/content/docs/workers/tutorials/using-prisma-postgres-with-workers/index.mdx

```diff
@@ -0,0 +1,225 @@
+
+In this tutorial, you will learn how to:
+
+- Set up a Cloudflare workers project with [Prisma ORM](https://www.prisma.io/docs).
+- Create a Prisma Postgres instance from the CLI.
+- Query the database from Workers.
+- Deploy the worker to Cloudflare.
+
+## Prerequisites
+
+To follow this guide, ensure you have the following:
+
+- Node.js `v18.18` or higher installed.
+- An active [Cloudflare account](https://dash.cloudflare.com/).
+- A basic familiarity with installing and using command-line interface (CLI) applications.
+
+## 1. Create a new Worker project
+
+Begin by using [C3](/pages/get-started/c3/) to create a Worker project in the command line:
+
+```sh
+npm create cloudflare@latest prisma-postgres-worker -- --type=hello-world --ts=true --git=true --deploy=false
+```
+
+Then navigate into your project:
+
+```sh
+cd ./prisma-postgres-worker
+```
+
+Your initial `src/index.ts` file contains a simple request handler:
+
+```ts title="src/index.ts"
+export default {
+	async fetch(request, env, ctx): Promise<Response> {
+		return new Response("Hello World!");
+	},
+} satisfies ExportedHandler<Env>;
+```
+
+## 2. Setup Prisma in your project
+
+In this step, we set up Prisma ORM with a Prisma Postgres database using the CLI. Then we'll create and execute helper scripts to create tables in our database and generate a Prisma client to query it.
+
+### 2.1. Install required dependencies
+
+Install Prisma CLI as a dev dependency:
+
+```sh
+npm install prisma --save-dev
+```
+
+Install the [Prisma Accelerate client extension](https://www.npmjs.com/package/@prisma/extension-accelerate) as that's required for Prisma Postgres:
+
+```sh
+npm i @prisma/extension-accelerate
+```
+
+Install the [`dotenv-cli` package](https://www.npmjs.com/package/dotenv-cli) to load environment variables from `.dev.vars`:
+
+```sh
+npm i -D dotenv-cli
+```
+
+### 2.2. Create a Prisma Postgres database and initialize Prisma
+
+Initialize Prisma in your application:
+
+```sh
+npx prisma@latest init --db
+```
+
+If you don't have a [Prisma Data Platform](https://console.prisma.io/) account yet, or if you are not logged in, the command will prompt you to log in using one of the available authentication providers. A browser window will open so you can log in or create an account. Return to the CLI after you have completed this step.
+
+Once logged in (or if you were already logged in), the CLI will prompt you to select a project name and a database region.
+
+After the command runs successfully, it will create:
+
+- A project in your [Platform Console](https://console.prisma.io/) containing a Prisma Postgres database instance.
+- A `prisma` folder containing `schema.prisma`, where you will define your database schema.
+- An `.env` file in the project root, which will contain the Prisma Postgres database url `DATABASE_URL=<your-prisma-postgres-database-url>`.
+
+You will use a `.dev.vars` file instead of the created `.env` file.
```

Comment:
```suggestion
Note that Cloudflare Workers don't support `.env` files. You will use a file called `.dev.vars` instead of the `.env` file that was just created.
```